logdna/logdna.py

```diff
@@ -45,6 +45,7 @@ def __init__(self, key, options={}):
         self.retry_interval_secs = options.get('retry_interval_secs', defaults['RETRY_INTERVAL_SECS'])
         self.tags = options.get('tags', [])
         self.buf_retention_byte_limit = options.get('buf_retention_limit', defaults['BUF_RETENTION_BYTE_LIMIT'])
+        self.user_agent = 'python/%s' % (options.get('user_agent', defaults['USER_AGENT']))
```

Comment:
@vilyapilya, still we can't expect the customers to pass just version of the library they build on top of this `python` code library - let's say at some point we build `ansible` library and we wanna use this `python` code library and do proper versioning; so, if we pass only the version in `options['user-agent']` field, it's gonna be `python/<version>` instead of `ansible/<version>` - we need to provide more flexibility

Reply:
gotcha